test/fakeOrigin/build/build_rpm.sh

```diff
@@ -70,7 +71,7 @@ cp $BUILDDIR/RPMS/x86_64/*.rpm ./dist/
 # Cross compile because we can
 GOBINEXT=""
 for GOOS in darwin linux windows; do
-  for GOARCH in 386 amd64; do
+  for GOARCH in amd64; do
```

Comment:
is this removing architecture support?

Reply:
There was a build error when trying to build the darwin/386 GOOS/GOARCH pair because Go does not support that pair (766e5aa1b0). I removed it so that fakeOrigin's build_rpm.sh would exit successfully.

Do we care about 32-bit support? If so, we could just skip over the darwin/386 pair.

Reply:
I think that would be better. I don't think there's anyone trying to do anything with this on 32-bit machines, but then again I'd reasonably expect to be able to run a lightweight fake origin on my raspi from high school, so who knows. If it's easy to not drop support for something valid, I say we do it.

Reply:
Re-added 386 GO_ARCH but skipping darwin/386 in 8551cbbf7c